### PR TITLE
add a generic send keys methods for unity as it can't access elements

### PR DESCRIPTION
**Pull request by rich-bugsnag**
add a generic send keys methods for unity as it can't access elements

### CODE REVIEW

File: lib/maze/driver/appium.rb [hunk 1]
```diff
@@ -112,6 +112,17 @@ def set_rotation(orientation)
...
       end
 
+      # Send keys to the device without a specific element
+      #
+      # @param text [String] the text to send
+      def send_keys(text)
+        @driver.send_keys(text)
+      end
+
+      def hide_keyboard
```
Comment (twometresteve) [inline lib/maze/driver/appium.rb:122]: nit: add a comment above the method

**Comment by twometresteve** [comment]
Looks good, but please add an entry to `CHANGELOG.md` (add a new section for 5.8.0 with date TBD and list this as an enhancement).

File: lib/maze/driver/appium.rb (hunk 1 above)
Comment (rich-bugsnag) [reply lib/maze/driver/appium.rb]: i remoed this method, i don't need it in the end

**Review by twometresteve — APPROVED** [review]
LGTM

**Pull request merged**